### PR TITLE
chore(build): add the windows integration in travis

### LINKED ISSUE #888: Use the travis Windows integration instead of Appveyor
Windows is now supported on Travis, it would be simpler to ditch Appveyor and have only one configuration. It also seems the case that Appveyor isn't building since some times ago.

https://blog.travis-ci.com/2018-10-11-windows-early-release

### PR DESCRIPTION
**Pull request by zarov**
Seems like it works correctly now.

Fixes #888 

Before merging though: do we really need the Windows integration ? It greatly increases the waiting time when modifying things in a PR.

### CODE REVIEW

**Comment by gchoqueux** [comment]
May be use test node Current and add window test.

**Comment by zarov** [comment]
Done !

**Comment by gchoqueux** [comment]
could you remove the `LTS linux `node?

**Comment by zarov** [comment]
Done

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
-  - "node"
+
+matrix:
+  include:
+    - os: linux
+      node_js: "node"
+    - os: windows
+      node_js: "node"
+
 after_success:
```
Comment (gchoqueux) [inline .travis.yml:10]: you need to add test in` autodeploy.sh` to deploy only on linux os.

Comment (zarov) [reply .travis.yml:10]: Well seen

**Comment by gchoqueux** [comment]
Unfortunately, windows tests takes double time.

**Comment by gchoqueux** [comment]
integration migrated in github action

**Pull request closed without merging**